base/db/tests/testthat/test-get.trait.data.pft.R

```diff
@@ -1,8 +1,12 @@
-
 context("get.trait.data.pft")
 
-con <- db.open(
-  params = list(user = "bety", password = "bety", host = "localhost"))
+con <- tryCatch(db.open(params = list(
+  user = "bety",
+  password = "bety",
+  host = "localhost"
+)), error = function(e) {
+  skip("Skipping tests because unable to establish database connection.")
+})
```

Comment:
My original goal here was to make it as easy as possible to run tests locally, without needing to worry about setting up BETY. In theory, this test should still run on Travis, since that does create a database connection, so it should still catch any errors.

But, point taken that skipping too freely isn't ideal. I would feel better about removing the `skips` if the database connection in question wasn't hard-coded to `localhost` (for instance, if it was controlled by `options` and defaulted to `localhost` if unset). That's a bit more involved, though, so I can drop these skips in the meantime.

Reply:
Addressed in https://github.com/PecanProject/pecan/pull/2273/commits/102c129a4a76465f275e807392d02539922fdc76.